src/error.rs

```diff
@@ -0,0 +1,79 @@
+#![allow(dead_code)]
```

Comment:
This should no longer be necessary?
(If it's still necessary it should be put on something more specific than a module.)